### ISSUE TITLE
Pasting values for environment variables fails when creating a Deployment via Cluster Explorer

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#4965]
Internal reference: SURE-3682
Reported in 2.5.11

Issue description:
When creating a Deployment via Cluster Explorer, pasting values for Environment Variables always fails the first time

Business impact:
UI usage is impaired when you always have to workaround this bug when working with the UI

Repro steps:

- Go to Cluster Explorer, Create a new Deployment
- At "Environment Variables", click "Add Variable"
- Keep default ("Key/Value Pair") and try to paste some value from your clipboard with CTRL+V 

The variable field disappears immediately. When adding the variable a second time, it works.

Actual behavior:
Paste works the first time and the field doesn't disappear

Expected behavior:
The field disappears the first time

Sounds related to another fix https://github.com/rancher/dashboard/issues/4196 though that work was done on file import, not copy & paste.

**Comment by nwmac** [comment]
Fixed in at least 2.6.2

**Comment by Auston-Ivison-Suse** [comment]
**Setup For Reproduction**
- Rancher Version: v2.5.11

Reproduction steps: 
1.Go to Cluster Explorer, Create a new Deployment
2.At "Environment Variables", click "Add Variable"
3. Keep default ("Key/Value Pair") and try to paste some value from your clipboard with CTRL+V

**Setup For validation**
-Rancher Version: v2.6-head(7efbeab)
1.Go to Cluster Explorer, Create a new Deployment
2.At "Environment Variables", click "Add Variable"
3. Keep default ("Key/Value Pair") and try to paste some value from your clipboard with CTRL+V

**Notes**

I was only able to reproduce this in the cluster explorer on a v2.5.11 rancher setup. 

I tried reproducing the issue on v2.6.3, v.2.6.2 with no luck. 


**Sanity Check**
I validated the issue didn't appear in the following types of workloads.
- CronJobs
-Jobs
-StatefulSets
-Pods
-DaemonSets
